### ISSUE TITLE
xref in webwork causes failure instead of giving unresolved message

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#1146]
Add the following to the minimal webwork example, in the statement:

`<p><xref ref="something"/></p>`

where "something" is literal, an obviously the target does not exist. Then do "make mini-extraction" and it fails. I get:

```
runtime error: file /Users/alex.jordan/mathbook/xsl/mathbook-common.xsl line 8336 element call-template
```

I'd hope to get some kind of "unresolved xref" come through. This is a practical issue when commenting out chunks of a book and a webwork has an xref pointing into what is commented out.

**Comment by rbeezer** [comment]
Line 8336 (and following) call "abstract" templates:
```xsl
<xsl:template name="inline-warning">
<xsl:template name="margin-warning">
```
which need to be defined in some conversion (the extraction?).

You can see definitions in the LaTeX and HTML conversions.  Perhaps the easiest thing to do is to define them to do nothing, since they only react to the `$author-tools` variable.

This is really old code which could use a facelift, but I've got no energy for that now.  So a simple PR that eases your pain might be the best route now.

**Comment by rbeezer** [comment]
Resolved at #1147
